src/em/config/em_configuration.cpp

```diff
@@ -1021,7 +1021,7 @@ int em_configuration_t::handle_topology_response(unsigned char *buff, unsigned i
 
         } else {
             found_profile = true;
-			memcpy(&profile, tlv->value, tlv->len);
+			memcpy(&profile, tlv->value, ntohs(tlv->len));
```

Comment:
It seems be quick fix. Unfortunately in the code I see to tlv->len is set variable converted by using htons function

Reply:
link to jira with more detialed description https://jira.rdkcentral.com/jira/browse/RDKBWIFI-12

Reply:
Not to merge yet. To verify one thing from my side.